torch/_subclasses/functional_tensor.py

```diff
@@ -1,7 +1,7 @@
 import contextlib
 import warnings
 from abc import ABC, abstractmethod
-from typing import Any, Callable, ContextManager, Dict, Optional, Tuple
+from typing import Any, Callable, ContextManager, Dict, Optional, Tuple, Union
```

Comment:
I should remove all the type hinting and add to diff to make this PR easier to review